packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react/remotesignout.md

```diff
@@ -0,0 +1,53 @@
+After performing [local signout](/docs/guides/sign-users-out/react/sign-out-of-your-app/), navigate the user's browser to the [OIDC logout page](https://developer.okta.com/docs/reference/api/oidc/#logout).
+
+This page will clear the user's Okta session then redirect back to the `post_logout_redirect_uri` provided. This URI must be one of those listed in the `Logout redirect URI` section of your application's settings.
```

Comment:
This page clears the user's Okta session, and then redirects back to the `post_logout_redirect_uri` that is provided.